src/common/ui-components/HeroBanner/HeroBanner.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import { Koros } from 'hds-react';
+
+import { Video } from '../Video/Video';
+import styles from './HeroBanner.module.scss';
+import ImageWithCard from '../ImageWithCard/ImageWithCard';
+const cx = classNames.bind(styles);
+
+type HeroBannerProps = React.PropsWithChildren<{
+  src?: string;
+  hasVideo?: boolean;
+  className?: string;
+}>;
+
+const HeroBanner: React.FC<HeroBannerProps> = ({
+  src,
+  children,
+  hasVideo,
```

Comment:
> hasVideo

Unused? Use a video url instead? Commented on the `Video` component.

Reply:
This is for the future use, in order to determine the type of video or if the video is shown at all.

Reply:
Yes, but it's a simple addition to this. For example by changing it to a string:

> { videoUrl && <Video url={videoUrl} /> }

Reply:
that sounds good

Reply:
done

Reply:
`hasVideo` is still present and unused, I would rather have that be replaced with `videoUrl`. After that this PR looks good 👍